docs/ref/templates/builtins.txt

```diff
@@ -285,6 +285,11 @@ Or if only some variables should be escaped, you can use::
 
     {% firstof var1 var2|safe var3 "<strong>fallback value</strong>"|safe %}
 
+
```

Comment:
Add `.. versionadded:: 1.9` and a mention in the 1.9 release notes.
Also, only 1 blank line between sections please.